bless/config/bless_lyft_config.py

```diff
@@ -0,0 +1,77 @@
+"""
+.. module: bless.config.bless_config
+    :copyright: (c) 2016 by Netflix Inc., see AUTHORS for more
+    :license: Apache, see LICENSE for more details.
+"""
+import configparser
+
+from bless.config.bless_config import BLESS_OPTIONS_SECTION
+
+CERTIFICATE_VALIDITY_WINDOW_SEC_OPTION = 'certificate_validity_seconds'
+CERTIFICATE_VALIDITY_SEC_DEFAULT = 60 * 2
+
+ENTROPY_MINIMUM_BITS_OPTION = 'entropy_minimum_bits'
+ENTROPY_MINIMUM_BITS_DEFAULT = 2048
+
+RANDOM_SEED_BYTES_OPTION = 'random_seed_bytes'
+RANDOM_SEED_BYTES_DEFAULT = 256
+
+CROSS_ACCOUNT_ROLE_ARN_OPTION = 'cross_account_role_arn'
+CROSS_ACCOUNT_ROLE_ARN_DEFAULT = None
+
+LOGGING_LEVEL_OPTION = 'logging_level'
+LOGGING_LEVEL_DEFAULT = 'INFO'
+
+CERTIFICATE_TYPE_OPTION = 'certificate_type'
+CERTIFICATE_TYPE_DEFAULT = 'user'
+
+BLESS_CA_SECTION = 'Bless CA'
+CA_PRIVATE_KEY_FILE_OPTION = 'ca_private_key_file'
+KMS_KEY_ID_OPTION = 'kms_key_id'
+
+KMSAUTH_KEY_ID_OPTION = 'kmsauth_key_id'
+KMSAUTH_KEY_ID_DEFAULT = None
+
+KMSAUTH_CONTEXT_OPTION = 'kmsauth_context'
+KMSAUTH_CONTEXT_DEFAULT = None
+
+REGION_PASSWORD_OPTION_SUFFIX = '_password'
+
+
+class BlessLyftConfig(configparser.RawConfigParser, object):
```

Comment:
Unfortunatley, Netfllix have altered the way configs work significantly in the past 4 years. This is just mostly lifted from the old files and modernized, but we should eventually refactor to take advantage of the netflix commits